### ISSUE TITLE
[Feature]: Use Utils in Templates

### ISSUE THREAD

**Issue by janstadt** [piitaya/lovelace-mushroom#1145]
### Requirements

- [X] I have updated Mushroom to the latest available version
- [X] I did a search to see if there is a similar issue or if a pull request is open.

### Is your feature request related to a problem?

Im just curious if there is a way to use some of your utility functions in templates? I'd like to use `getWeatherStateSVG` in a template chip so i dont have to duplicate that functionality, but basically im looking for the ability to essentially extend the weather chip so that i can also add air quality from the IQVIA integration.

### Describe the solution you'd like

An example or documentation on how to use some of your public utility functions and/or an example of extending a chip to include additional information.

**Comment by piitaya** [comment]
I don't think so. Template are rendered by core, not by front-end. There is ton of issues of people that want template support for all others cards but it's a huge work... so I don't think it will happen with the current Mushroom architecture.
